Formula/utf8proc.rb

```diff
@@ -3,6 +3,7 @@ class Utf8proc < Formula
   homepage "https://juliastrings.github.io/utf8proc/"
   url "https://github.com/JuliaStrings/utf8proc/archive/v2.5.0.tar.gz"
   sha256 "d4e8dfc898cfd062493cb7f42d95d70ccdd3a4cd4d90bec0c71b47cca688f1be"
+  license "MIT AND Unicode-DFS-2015"
```

Comment:
```suggestion
  license all_of: ["MIT", "Unicode-DFS-2015"]
```

I think you're right about `Unicode-DFS-2015` but I want to get a second opinion. The license listed in the [`LICENSE.md`](https://github.com/JuliaStrings/utf8proc/blob/master/LICENSE.md#unicode-data-license) links to [this license](http://www.unicode.org/copyright.html) which matches [`Unicode-DFS-2015`](https://spdx.org/licenses/Unicode-DFS-2015.html) but these first few lines are missing from the `LICENSE.md` file:

```
Unicode Data Files include all data files under the directories http://www.unicode.org/Public/, http://www.unicode.org/reports/, and http://www.unicode.org/cldr/data/. Unicode Data Files do not include PDF online code charts under the directory http://www.unicode.org/Public/. Software includes any source code published in the Unicode Standard or under the directories http://www.unicode.org/Public/, http://www.unicode.org/reports/, and http://www.unicode.org/cldr/data/.

NOTICE TO USER: Carefully read the following legal agreement. BY DOWNLOADING, INSTALLING, COPYING OR OTHERWISE USING UNICODE INC.'S DATA FILES ("DATA FILES"), AND/OR SOFTWARE ("SOFTWARE"), YOU UNEQUIVOCALLY ACCEPT, AND AGREE TO BE BOUND BY, ALL OF THE TERMS AND CONDITIONS OF THIS AGREEMENT. IF YOU DO NOT AGREE, DO NOT DOWNLOAD, INSTALL, COPY, DISTRIBUTE OR USE THE DATA FILES OR SOFTWARE.
```

Reply:
IANAL, but...

The first paragraph specifies a non-exclusive list of data file locations that don't apply in this case, so I'm not surprised the Unicode folks struck it out.

As for the second paragraph, it's a warning that "use implies acceptance", but since that's already a given in the software licensing world, the Unicode folks may have removed it as being unnecessary.

In any case, the terms of the license (which form the legal "meat") fall below the copyright notice. As long as they match `Unicode-DFS-2015`, I think it's safe to claim it is such.